### DISCUSSION TITLE
Server fails to register with directory server if IPv6 enabled

### DISCUSSION THREAD

**Discussion by rdica** [jamulussoftware/jamulus#2078] [Technical Support & Help]
Upon building 3.8.1rc2 and restarting my server, it fails to register with `--centralserver anygenre2.jamulus.io:22224`. I then reverted to previous 3.8.1rc1 binary which was working fine until now, and it too fails to register.
Has something changed with the directory servers?

**Comment by rdica** [comment]
I have been enabling IPv6 since rc1 and this has worked fine until sometime since Saturday (my last server restart). Removing the `-6` argument seems to have fixed things. Why is this failing now?

**Reply by rdica** [reply]
And now it's working again.

**Reply by softins** [reply]
My guess is a temporary problem with the directory server, or with the route to it. Since a DNS lookup of `anygenre2.jamulus.io` only returns an IPv4 address, it will make no difference whether your server has IPv6 enabled or not.

**Reply by rdica** [reply]
I forgot to mention I'd tried anygenre1, 2, and 3 with their respective port numbers before I tried removing the `-6` arg. In any case I was able to get the server registered again and played several hours last night with a direct IPv6 connection. Thanks!

**Reply by rdica** [reply]
Also, I've unmarked the answer since disabling ipv6 is never a fix and it's unknown what the actual problem was. Since I can now run the server with IPv6 enabled, there was nothing to fix. Thanks again!
